app/test_split_integer.py

```diff
@@ -2,20 +2,44 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    number_of_parts = 6
+    parts = split_integer(value, number_of_parts)
+
+    assert sum(parts) == value, "Sum of parts is not equal to the value!"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 15
+    number_of_parts = 5
+    parts = split_integer(value, number_of_parts)
+
+    assert parts.count(parts[0]) == number_of_parts, "Not all parts are equal!"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    value = 8
+    number_of_parts = 1
+    parts = split_integer(value, number_of_parts)
+
+    assert sum(parts) == value, (
+        "Part is not equal to the value if number of parts == 1!"
+    )
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    value = 17
+    number_of_parts = 4
+
+    assert split_integer(value, number_of_parts) == [4, 4, 4, 5], (
```

Comment:
The test `test_parts_should_be_sorted_when_they_are_not_equal` is missing the call to `split_integer` to obtain the `parts` variable. You should call `split_integer(value, number_of_parts)` and assign it to `parts` before the assertion.